src/consumption.cpp

```diff
@@ -1384,29 +1384,29 @@ void Character::modify_morale( item &food, const int nutr )
         const int minor_morale_penalty = -10;
         const int minor_morale_bonus = 10;
         const int maximum_stacked_morale_penalty = -400;
-        if( cannibal && psycho ) {
-            add_msg_if_player( m_good,
-                               _( "You worry that your hunger for human flesh is going to be a liability one of these days." ) );
-            // Reduced penalties
-            add_morale( morale_cannibal, minor_morale_penalty, minor_morale_penalty * 5, 6_hours, 3_hours );
+        if( sapiovore ) {
+            add_msg_if_player( _( "Mmh.  Tastes like venison." ) );
+        } else if( sapiovore && spiritual ) {
```

Comment:
You must check for the combination of sapiovore and spiritual before sapiovore. This code is unreachable